os/board/common/Kconfig

```diff
@@ -134,3 +134,216 @@ config AUTOMOUNT_ROMFS
 	depends on FS_ROMFS
 	---help---
 		If enabled, mount romfs partition automatically at boot.
+
+config BOARD_CRASHDUMP
```

Comment:
I think this and belows are in os/board/Kconfig. I think they are duplicated. Could you confirm?